src/main/java/uni/miskolc/ips/ilona/measurement/persist/mysql/entity/ZoneEntity.java

```diff
@@ -0,0 +1,16 @@
+package uni.miskolc.ips.ilona.measurement.persist.mysql.entity;
+
+import lombok.*;
+import javax.persistence.*;
+
+@Entity
+@Getter
```

Comment:
Data annotation instead of Getter Setter

https://projectlombok.org/features/Data

Add Builder annotation as well.

Reply:
Done